examples/custom-indexer/rust/README.md

```diff
@@ -21,13 +21,7 @@ cargo run --bin remote_reader
 ### Local Reader example
 The local reader example saves progress in a file called `/tmp/local_reader_progress` and monitors checkpoint files in the `chk` directory
```

Comment:
There was a callout earlier about the chk files not being in (the expected) BCS format. Is there a good spot in the readme to call attention to this?